pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/AuthenticationTest.java

```diff
@@ -1,110 +1,182 @@
 package org.broadinstitute.dsm;
 
-import static org.broadinstitute.dsm.TestHelper.cfg;
-import static org.broadinstitute.dsm.TestHelper.setupDB;
 
 import org.broadinstitute.dsm.db.dao.user.UserDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
 import org.broadinstitute.dsm.db.dto.user.UserDto;
 import org.broadinstitute.dsm.model.NameValue;
 import org.broadinstitute.dsm.model.patch.Patch;
+import org.broadinstitute.dsm.service.admin.SetUserRoleRequest;
+import org.broadinstitute.dsm.service.admin.UserAdminService;
+import org.broadinstitute.dsm.service.admin.UserAdminServiceTest;
 import org.broadinstitute.dsm.statics.DBConstants;
 import org.broadinstitute.dsm.util.UserUtil;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.*;
 
-public class AuthenticationTest {
-    public static final String UNIT_TESTER_EMAIL = "testUsersEmails.unitTesterEmail";
-    public static final String GP_UNIT_TESTER_EMAIL = "testUsersEmails.gpUnitTesterEmail";
+import java.util.*;
 
+import static org.broadinstitute.dsm.service.admin.UserAdminService.USER_ADMIN_ROLE;
+import static org.broadinstitute.dsm.statics.DBConstants.KIT_SHIPPING;
+import static org.broadinstitute.dsm.statics.DBConstants.PT_LIST_VIEW;
 
-    @Before
-    public void first() {
-        setupDB();
+public class AuthenticationTest extends DbTxnBaseTest {
 
+    static UserDto dsmAdminUser;
```

Comment:
These should be private. Are you seeing lint warnings for this in intelliJ?

Reply:
I'm not seeing any warnings.  Will change to private.